openslides/users/static/js/users/site.js

```diff
@@ -999,6 +1017,8 @@ angular.module('OpenSlidesApp.users.site', ['OpenSlidesApp.users'])
 .config([
     'gettext',
     function (gettext) {
+        // string for "core"
```

Comment:
I do not understand this string at this place here. Why to defining "Core" in core?